### PR TITLE
pfs_integration_test: Update to run with calibId and constructArc

### PR DESCRIPTION
**Pull request by ziggyman**

### CODE REVIEW

File: bin/pfs_integration_test.sh [hunk 1]
```diff
@@ -88,33 +88,43 @@ ingestImages.py $TARGET --mode=link \
 # Build bias
 constructBias.py $TARGET --rerun $RERUN/bias \
     --id dateObs=2015-12-22 arm=r spectrograph=1 \
+    --calibId calibDate=2015-12-22 arm=r spectrograph=1 \
```
Comment (PaulPrice) [inline bin/pfs_integration_test.sh]: I'm surprised you need `calibDate`. I thought it should be calculated automatically.

File: bin/pfs_integration_test.sh [hunk 2]
```diff
@@ -88,33 +88,43 @@ ingestImages.py $TARGET --mode=link \
...
 genCalibRegistry.py --root $TARGET/CALIB --validity 1000
 ( $CLEANUP && rm -r $TARGET/rerun/$RERUN/flat ) || true
 
 # Process an arc
-detrend.py $TARGET --rerun $RERUN/detrend --id visit=103
-reduceArc.py $TARGET --rerun $RERUN/detrend --id visit=103
+constructArc.py $TARGET --rerun $RERUN/arc --id visit=103 \
+    --calibId calibDate=2016-11-11 arm=r spectrograph=1 \
+    -c isr.doBias=True isr.doDark=True isr.doFlat=True \
+    isr.doLinearize=False --batch-type none &&
```
Comment (PaulPrice) [inline bin/pfs_integration_test.sh]: Are these overrides something that you want every time you run `constructArc.py`? If so, they should go in the obs package or in the definition of `constructArc.py`.

File: bin/pfs_integration_test.sh [hunk 3]
```diff
@@ -88,33 +88,43 @@ ingestImages.py $TARGET --mode=link \
...
+genCalibRegistry.py --root $TARGET/CALIB --validity 1000 &&
+( $CLEANUP && rm -r $TARGET/rerun/$RERUN/arc ) || true
+
+reduceArcRefSpec.py $TARGET --rerun $RERUN/arc --id visit=103
```
Comment (PaulPrice) [inline bin/pfs_integration_test.sh]: Clean up `$TARGET/rerun/$RERUN/arc`?

**Comment by ziggyman** [comment]
merged into master as tickets/PIPE2D-66

**Pull request closed without merging**